static/app/components/replays/breadcrumbs/breadcrumbItem.tsx

```diff
@@ -338,7 +274,6 @@ const CrumbItem = styled(PanelItem)<{isErrorFrame?: boolean}>`
 `;
 
 const CodeContainer = styled('div')`
-  margin-top: ${space(1)};
```

Comment:
I used `<Flex>` above to group the title+description, which means I can remove the margin and use `gap` instead for anything that sits below the description 👍